tigerpath/majors_and_certificates/degrees/BSE_2018.json

```diff
@@ -60,16 +57,15 @@
       "name": "Mathematics",
       "max_counted": 1,
       "min_needed": 4,
-      "description": "Mathematics (four semesters)",
-      "explanation": null,
+      "explanation": "<b>Mathematics</b><br><i>four semesters</i>",
       "pdfs_allowed": false,
+      "double_counting_allowed": true,
```

Comment:
This is here just for consistency with the other prerequisites. It doesn't make a difference here either way.